test/test_hct.py

```diff
@@ -23,18 +23,19 @@ def phys_cell(request):
 def make_unisolvent_points(element):
     degree = element.degree()
     ref_complex = element.get_reference_complex()
-    sd = ref_complex.get_spatial_dimension()
     top = ref_complex.get_topology()
     pts = []
-    for cell in top[sd]:
-        pts.extend(ref_complex.make_points(sd, cell, degree+sd+1))
+    for dim in top:
+        for entity in top[dim]:
+            pts.extend(ref_complex.make_points(dim, entity, degree, variant="gll"))
     return pts
 
 
```

Comment:
Again, a harness for testing mass matrix conditioning on one cell (as a function of h) would be good.  You can do this in a separate file and cover Hermite/Argyris/HCT/Morley all in one test.